src/main/java/org/jitsi/videobridge/octo/OctoRelay.java

```diff
@@ -19,7 +19,8 @@
 import org.jitsi.rtp.*;
 import org.jitsi.rtp.rtp.*;
 import org.jitsi.utils.*;
-import org.jitsi.utils.logging.*;
+import org.jitsi.utils.collections.*;
+import org.jitsi.utils.logging2.*;
```

Comment:
Huh, we still had old logger in jvb in some places? I thought you'd switched everything over.  Are there any other places like this?

Reply:
yes, looks like there are.  i'll do a final sweep of this soon.